clients/privacy-center/pages/consent.tsx

```diff
@@ -11,179 +11,260 @@ import {
   useToast,
 } from "@fidesui/react";
 import { useRouter } from "next/router";
-import {
-  ConfigErrorToastOptions,
-  ErrorToastOptions,
-  SuccessToastOptions,
-} from "~/common/toast-options";
+import { ErrorToastOptions, SuccessToastOptions } from "~/common/toast-options";
```

Comment:
Side-by-side diff will be your friend for reviewing this. I now have each useEffect block commented, with each handling a single response situation.